chore: add instruction for migrate JupyterLab Classic to Current (#50)
ibm-skills-network/author-workbench-knowledgebase: docs/labs/jupyterlab-filelibrary.md

> # Acess File Library within Author Workbench

Reviewer: ```suggestion
# Access File Library within Author Workbench
```

Follow these steps to open file library in Author Workbench

> 1. **Open Your Lab:** Begin by launching your lab detail page.

Reviewer: ```suggestion
1. **Open Your Lab:** Begin by clicking on your lab in Author Workbench to view its details page.
```

> 2. **Locate the Content Tab:** In the lab, find and click on the "Content" Tab. Refer to the image below for guidance:

Reviewer: ```suggestion
2. **Locate the Content Tab:** In the lab, find and click on the "Content" Tab. Refer to the image below for guidance.
```
